pylot/perception/detection/detection_operator.py

```diff
@@ -73,14 +77,21 @@ def __init__(self, camera_stream: erdos.ReadStream,
 
         logger.debug('\n\n*********Path of the model using now: %s', model_path)
 
-        self._coco_labels = load_coco_labels(self._flags.path_coco_labels)
+        self._model_names = self._flags.obstacle_detection_model_names
+        if 'yolo' in self._model_names:
+            self._coco_labels = load_coco_labels(self._flags.path_coco_labels, is_zero_index=True)
+        else:
+            self._coco_labels = load_coco_labels(self._flags.path_coco_labels)
```

Comment:
Pylot assumes all object detectors will output class IDs that are 1-indexed. However, keras-YOLOv3-model-set trains the model with classes being 0-indexed.